### PR TITLE
Pagination and Tables

### PR DESCRIPTION
**Pull request by EliasBoulharts**

### CODE REVIEW

File: project-static/squest/css/squest.css [hunk 1]
```diff
@@ -12,3 +12,7 @@
     cursor: pointer;
     position: relative;
 }
+
+.table thead tr {
```
Comment (Sispheor) [inline project-static/squest/css/squest.css:16]: Risky as it will change all our table.

File: service_catalog/views/common.py [hunk 2]
```diff
@@ -277,16 +276,6 @@ def instance_support_details(request, instance_id, support_id, breadcrumbs):
     return render(request, "service_catalog/common/instance-support-details.html", context)
 
 
-@login_required
-def request_list(request):
-    if request.user.is_superuser:
-        f = RequestFilter(request.GET, queryset=Request.objects.all())
-        return render(request, 'service_catalog/admin/request/request-list.html', {'filter': f})
```
Comment (Sispheor) [inline service_catalog/views/common.py:284]: Customer and admin page need to be implemented back

File: templates/custom_columns/request_operation_type.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<h4>
```
Comment (Sispheor) [inline templates/custom_columns/request_operation_type.html:1]: Need to be placed in the sub folder service catalog

File: profiles/filters/billing_group_filter.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import django_filters
+from django import forms
+
+from profiles.models import BillingGroup
+from utils.squest_filter import SquestFilter
+
+
+class BillingGroupFilter(SquestFilter):
+    name = django_filters.CharFilter(field_name='name', lookup_expr='exact',
```
Comment (a-belhadj) [inline profiles/filters/billing_group_filter.py:9]: remove lookup_expr already in init

File: profiles/views/user_by_billing_group_list_view.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+
+class UserByBillingGroupTable(tables.Table):
+    actions = TemplateColumn(template_name='custom_columns/user_by_group_actions.html', orderable=False)
+
+    class Meta:
+        model = User
+        attrs = {"id": "user_by_billing_group_table", "class": "table squest-pagination-tables "}
+        fields = ("username", "email", "actions")
+        template_name = "django_tables2/bootstrap4.html"
```
Comment (Sispheor) [inline profiles/views/user_by_billing_group_list_view.py:18]: to be refactored

File: service_catalog/filters/global_hook_filter.py [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        model = GlobalHook
+        fields = ['name', 'model']
+
+    model = django_filters.MultipleChoiceFilter(
+        choices=HookModel.choices,
+        widget=forms.SelectMultiple())
+
+    state_instance = django_filters.MultipleChoiceFilter(
+        label="Instance state",
+        method='add_state',
```
Comment (Sispheor) [inline service_catalog/filters/global_hook_filter.py:21]: get_instance_state

File: service_catalog/filters/global_hook_filter.py [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        choices=InstanceState.choices,
+        null_value=None,
+        widget=forms.SelectMultiple(attrs={'data-live-search': "true"}))
+
+    state_request = django_filters.MultipleChoiceFilter(
+        label="Request state",
+        method='get_state',
```
Comment (Sispheor) [inline service_catalog/filters/global_hook_filter.py:28]: get_request_state

File: service_catalog/migrations/0012_auto_20210907_1030.py [hunk 8]
```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.1.7 on 2021-09-07 08:30
+
+from django.db import migrations, models
```
Comment (Sispheor) [inline service_catalog/migrations/0012_auto_20210907_1030.py:3]: migration to be squashed

File: service_catalog/urls.py [hunk 9]
```diff
@@ -44,14 +54,15 @@
     views.edit_service_operation, name='edit_service_operation'),
     path('settings/catalog/service/<int:service_id>/operations/<int:operation_id>/survey/',
     views.service_operation_edit_survey, name='service_operation_edit_survey'),
-    path('settings/global_hook/', views.global_hook_list, name='global_hook_list'),
+    #TODO:
```
Comment (Sispheor) [inline service_catalog/urls.py:57]: todo?

**Review by Sispheor — APPROVED** [review]

**Pull request merged**